deploy/playbooks/roles/common/tasks/circus.yml

```diff
@@ -8,11 +8,17 @@
     - chaussette
 
 - name: ensure /etc/circus exists
-  file: path=/etc/circus state=directory
+  file:
+    path: /etc/circus
+    state: directory
+    owner: "{{ ansible_ssh_user }}"
   sudo: true
 
 - name: ensure /var/log/circus exists
-  file: path=/var/log/circus state=directory
+  file:
+    path: /var/log/circus
+    state: directory
+    owner: "{{ ansible_ssh_user }}"
```

Comment:
can you add a `recurse: true` so that if there is anything in there that somehow has a different owner this will fix it too?